otherlibs/stdlib_beta/int.ml

```diff
@@ -0,0 +1,319 @@
+(**************************************************************************)
```

Comment:
I think this module should be named something other than Int.  It's too confusing and error-prone to have two different modules named `Int` in the stdlib, even if one is in `stdlib_beta`.  One day we'd like all of this to be in the normal upstream stdlib!

Reply:
Changed to `Stdlib_beta.Int_wrapper`